src/constants.js

```diff
@@ -88,6 +101,17 @@ export const RUN_CLI_PARAMS ={
     },
     metric: METRIC_PARAM,
     all: ALL_PARAM,
+    /**
+     * the reason we set a throttling default is three-fold
+     * 1) throttling must be active to get firstCPUIdle and firstInteractive metrics
+     * 2) it more realistically simulates real world conditions
+     * 3) Good 3G is also the lighthouse default
+     */
+    throttle: {
+        alias: 'tn',
```

Comment:
Not every option needs an alias. I would be fine to keep only `--throttle`. Also, does it produce `--tn` or `-tn` as an alias? The latter is confusing as it might mean both `-t` and `-n` joined.

Reply:
That's a good point.  I guess we don't necessarily need an alias here.